docs/01-overview/main-areas/observability/README.md

```diff
@@ -6,8 +6,8 @@ title: What is Observability in Kyma?
 
 Kyma comes with tools that give you the most accurate and up-to-date monitoring, logging and tracing data.
 
-- [Prometheus](https://prometheus.io/) is the open source monitoring and alerting toolkit that provides the telemetry data. This data is consumed by different addons, including [Grafana](https://grafana.com/) for analytics and monitoring, and [Alertmanager](https://prometheus.io/docs/alerting/alertmanager/) for handling alerts.
-- For logging, Kyma uses [Loki](https://github.com/grafana/loki), a Prometheus-like log management system.
+- [Prometheus](https://prometheus.io/) is the open source monitoring and alerting toolkit that provides the telemetry data. This data is consumed by different addons, including [Grafana](https://grafana.com/) for analytics and monitoring, and [Alertmanager](https://prometheus.io/docs/alerting/alertmanager/) for handling alerts. By default, Prometheus stores up to 15 GB of data for a maximum period of 30 days. If the default size or time is exceeded, the oldest metrics are removed first.
```

Comment:
Maybe better write "the oldest records are removed first" to indicate that not the whole time series is deleted

Reply:
I took that phrase from the SKR doc :D Then we should change it there too 🙈

Reply:
I can live with both formulations. I was just wondering if people might consider the whole time series as the metric.

Reply:
```suggestion
- [Prometheus](https://prometheus.io/) is the open source monitoring and alerting toolkit that provides the telemetry data. This data is consumed by different addons, including [Grafana](https://grafana.com/) for analytics and monitoring, and [Alertmanager](https://prometheus.io/docs/alerting/alertmanager/) for handling alerts. By default, Prometheus stores up to 15 GB of data for a maximum period of 30 days. If the default size or time is exceeded, the oldest records are removed first.
```